taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = [
+            "license_number",
+        ]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "license number must be 8 characters long"
+            )
+        if (
+            not license_number[:3].isalpha()
+            or not license_number[:3].isupper()
+        ):
+            raise forms.ValidationError(
+                "First 3 characters must be uppercase letters"
+            )
+        if not license_number[3:].isdigit():
+            raise forms.ValidationError("Last 5 characters must be digits")
+        return license_number
+
+
+class DriverCreateForm(forms.ModelForm):

```

Comment:
The `DriverCreateForm` should inherit from `UserCreationForm` instead of `forms.ModelForm`. This is important to ensure that user creation is handled correctly with the built-in Django functionalities. Refer to the checklist: 'Use `UserCreationForm` while creating user.'